### PR TITLE
[INLONG-7720][Manager] Creating schema of StreamSource by Excel

### PR DESCRIPTION
**Pull request by featzhang**
#7720

### CODE REVIEW

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/ExcelTool.java [hunk 1]
```diff
@@ -0,0 +1,560 @@
...
+    /**
+     * Read the given input stream and return a list of objects
+     * @param is the input stream to read
+     * @param eClass the class of the objects to return
+     * @return a list of the objects
+     * @throws IOException if there is an error reading the input stream
+     * @throws IllegalAccessException if there is an error accessing the class
+     * @throws InstantiationException if there is an error instantiating the class
+     */
+    public static <E extends ExcelImportTemplate> List<E> read(InputStream is, Class<E> eClass)
```
Comment (healchow) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/ExcelTool.java:283]: This method is so long, could you please simply it by extracting some common methods?

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/ExcelTool.java [hunk 2]
```diff
@@ -0,0 +1,560 @@
...
+                if (excel != null) {
+                    String excelName = excel.name();
+                    fieldMap.put(excelName, field);
+                }
+            }
+
+            return fieldMap;
+        } else {
+            throw new IllegalArgumentException("It is not have fields in class '" + eClass.getCanonicalName() + "'");
+        }
```
Comment (healchow) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/ExcelTool.java:417]: In this case, you can simplify the method by using a guard statement(卫语句):

```java
if (fields.length <= 0) {
    throw new IllegalArgumentException();
}

// do your loop or other statements
```

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/ExcelTool.java [hunk 3]
```diff
@@ -0,0 +1,560 @@
...
+                cellvalue = cellvalue.replace("\\", "/");
+            }
+        } else {
+            cellvalue = "";
+        }
+
+        return cellvalue;
+    }
+
+    public static <E> List<Map<String, Object>> read2MapList(InputStream is, Class<E> eClass)
```
Comment (healchow) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/ExcelTool.java:438]: Too long method...

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/validator/ExcelRowValidator.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.common.tool.excel.validator;
+
+/**
+
```
Comment (healchow) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/validator/ExcelRowValidator.java:21]: Unecessay blank line.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/validator/ExcelCellValidator.java [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.common.tool.excel.validator;
+
+import java.io.Serializable;
+import org.apache.poi.hssf.usermodel.DVConstraint;
+
+/**
+
```
Comment (healchow) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/validator/ExcelCellValidator.java:24]: Unecessary blank line.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/validator/ExcelBatchValidator.java [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.common.tool.excel.validator;
+
+import java.util.List;
+
+/**
+
```
Comment (healchow) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/validator/ExcelBatchValidator.java:23]: Ditto

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/ExcelTool.java [hunk 7]
```diff
@@ -0,0 +1,560 @@
...
+import org.apache.poi.hssf.usermodel.HSSFSheet;
+import org.apache.poi.hssf.usermodel.HSSFWorkbook;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellType;
+import org.apache.poi.ss.usermodel.Row.MissingCellPolicy;
+import org.apache.poi.ss.util.CellRangeAddressList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ExcelTool {
```
Comment (vernedeng) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/tool/excel/ExcelTool.java:55]: Is there any open source Excel tools? 
Do we really need to implement excel tool?

**Comment by vernedeng** [comment]
I think it's better to divide this PR to two individual ones, one for excel tool, one for schema of StreamSource.

**Comment by dockerzhang** [comment]
Implemented by #7720 sub-tasks, so close it.

**Pull request closed without merging**